lms/static/sass/course/_student-notes.scss

```diff
@@ -45,6 +45,12 @@ $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
       @extend .content; // needed extend carried over from course handouts UI, but should be cleaned up
       width: 100%;
     }
+
+    .sr-is-focusable,
```

Comment:
Thanks, @clrux. I tested removing it and cherry-picking your commit in the other PR, and all looks good.